webhook/resources/backup/validator.go

```diff
@@ -51,5 +52,20 @@ func (b *backupValidator) Create(request *admission.Request, newObj runtime.Obje
 		return werror.NewInvalidError(fmt.Sprintf("BackupMode %v is not a valid option", backup.Spec.BackupMode), "")
 	}
 
+	// Check if backup target exists and is available
+	backupTargetName, ok := backup.Labels[types.LonghornLabelBackupTarget]
+	if !ok || backupTargetName == "" {
```

Comment:
NIT: You can directly use `backup.Labels[types.LonghornLabelBackupTarget] != ""` in Golang